ucm2/codecs/lpass/va-macro/DMIC0DisableSeq.conf

```diff
@@ -0,0 +1,5 @@
+DisableSequence [
+	cset "name='VA DMIC MUX0' ZERO"
+	cset "name='VA_DEC0 Volume' 0"
+	cset "name='VA_AIF1_CAP Mixer DEC0' 0"
+]
```

Comment:
The lpass directory is not so obvious for other people. Can you rename this to qcom-lpass or so?  Thank you.

Reply:
@perexg done